pympress/media_overlays/gst_backend.py

```diff
@@ -97,11 +97,24 @@ def mute(self, value):
         return False
 
 
-    def on_play(self, *args):
-        """ Start the scroll bar updating process.
-        """
+    def on_state_changed(self, bus, msg):
+        """ Triggers on_initial_play() when starting playback from being stopped. """
+        if msg.src == self.playbin:  # ignore the playbin's children
```

Comment:
I’d prefer `if != : return` so we limit nesting and can extend to other messages more easily if we ever need to